tests/build.gradle

```diff
@@ -34,9 +34,12 @@ tasks.withType(Test) {
 }
 
 dependencies {
-    compile "org.scala-lang:scala-library:${gradle.scala.version}"
-    compile "org.apache.openwhisk:openwhisk-tests:${gradle.openwhisk.version}:tests"
-    compile "org.apache.openwhisk:openwhisk-tests:${gradle.openwhisk.version}:test-sources"
+    implementation "junit:junit:4.11"
```

Comment:
Reason is here: https://github.com/apache/openwhisk-runtime-nodejs/pull/221#discussion_r936174947